docker-compose.yml

```diff
@@ -1,12 +1,22 @@
 version: '3'
 services:
+
     web:
-        build: ./
+        build: ./.docker/web
         container_name: svgtranslate_web
         ports:
           - ${TOOLFORGE_DOCKER_PORT}:80
         volumes:
-          - ./.docker/lighttpd.conf:/etc/lighttpd/lighttpd.conf:cached
+          - ./.docker/web/lighttpd.conf:/etc/lighttpd/lighttpd.conf:cached
```

Comment:
instead of mounting it, you could just add a `COPY` in `./.docker/web/Dockerfile`

unless you need to modify it without rebuilding?

Reply:
Good idea. Done.